ParabolInc/parabol
# feat(standups): Added start team prompt mutation

Fixes #5930 and #5931.

Test:
1. open new meeting screen
2. start async standup
3. empty view with a Parabol logo is displayed
4. close empty view by clicking the logo in the center of the screen
5. async standup meeting is displayed in the meetings view

![CleanShot 2022-03-18 at 15 46 36](https://user-images.githubusercontent.com/1017620/159325817-f3db1297-1951-42c6-9c42-b89fdbd2e236.gif)
